services/user.js

```diff
@@ -13,7 +13,6 @@ exports.getUserByEmail = async (email) => {
   try {
     const user = await User.findOne({
```

Comment:
tiene que traer el role.
ej. si el usuario es roleid 1
tiene que incluir un  campo 'role'  el cual tiene que traer el nombre  . en este caso  'Admin'.
tenes que resolver la consulta. me explico?

al devolverlo completo el controller lo va a tener disponible para agregarlo al token.

Reply:
incluir el roleId y el nombre del role en la respuesta de user

Reply:
const attributes = [
      'id',
      'firstName',
      'lastName',
      'password',
      'email',
      'image',
      [sequelize.col('role.name'), 'userRole'], // select 'role'.'name' from the JOIN
    ]
    const user = await User.findByPk(id, {
      attributes,
      include: {
        // includer other table
        model: Role, // model name
        as: 'role', // model alias
        attributes: [], // we don't want any atributes
      },
    })

Reply:
const attributes = [
      'id',
      'firstName',
      'lastName',
      'password',
      'email',
      'image',
      [sequelize.col('role.name'), 'userRole'], // select 'role'.'name' from the JOIN
    ]
    const user = await User.findByPk(id, {
      attributes,
      include: {
        // includer other table
        model: Role, // model name
        as: 'role', // model alias
        attributes: [], // we don't want any atributes
      },
    })

Reply:
Esto ultimo que me pasaste queres lo agregas vos o lo agrego yo en este pr?